docs/Contributing/Releasing-Fleet.md

```diff
@@ -17,6 +17,7 @@ Note: Please prefix versions with `fleet-v` (eg. `fleet-v4.0.0`) in git tags, He
    - [package.json](https://github.com/fleetdm/fleet/blob/main/tools/fleetctl-npm/package.json) (do not yet `npm publish`)
    - [Helm chart](https://github.com/fleetdm/fleet/blob/main/charts/fleet/Chart.yaml) and [values file](https://github.com/fleetdm/fleet/blob/main/charts/fleet/values.yaml)
    - [Terraform variables](https://github.com/fleetdm/fleet/blob/main/infrastructure/dogfood/terraform/aws/variables.tf)
+   - [Kubernetes manifest file](https://fleetdm.com/docs/deploying/server-installation#deploying-fleet-on-kubernetes)
```

Comment:
```suggestion
   - [Kubernetes `deployment.yml` example file](https://github.com/fleetdm/fleet/blob/main/docs/Deploying/Server-Installation.md#deploying-fleet-on-kubernetes)
```

Thanks for adding this link to the "Releasing Fleet" instructions! I have two suggested changes:
- What do you think about renaming the link to say "Kubernetes `deployment.yml` example file" instead of "Kubernetes manifest" file. 
  - As a contributor releasing Fleet, I didn't understand what part of the "Deploying Fleet on Kubernetes" I needed to update. With this updated name, I know what part of the doc page needs to be updated.
- Can you please update the URL to take the contributor to the exact markdown file? This way, the contributor knows what file needs to be updated when preparing a release of Fleet.